src/openforms/payments/contrib/ogone/tests/test_client.py

```diff
@@ -53,6 +53,23 @@ def test_payment_request_invalid_order_id(self):
 
         self.assertIn("Er is een fout opgetreden", payment_request.text)
 
+    def test_com_title_unicode_chars(self):
+        client = OgoneClient(self.merchant)
+
+        info = client.get_payment_info(
+            "xyz2024/OF-123456/987654321",
+            1000,
+            "http://foo.bar/return?bazz=buzz",
+            RETURN_ACTION_PARAM,
+            title="läääääääääääääämp! " * 10,  # 180 chars
```

Comment:
I think this is 190 characters because of the space at the end, also shouldn't title be truncated as well? Or is there no limit for the title?

Reply:
there's no limit for the title :) and counting is hard! will update the comment